### PR TITLE
Bump monitoring stack to latest stable versions

### LINKED ISSUE #2198: Bump ScyllaDBMonitoring to latest stable
### What should the feature do?

Versions used by monitoring stack are not latest stable, we should bump them to pick up latest dashboards and fixes.

### What is the use case behind this feature?

Have latest dashboards for ScyllaDB and all fixes coming with latest versions of monitoring stack.

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by zimnx**
**Description of your changes:**

Bumps following versions:
* ScyllaDB Monitoring - 4.8.3
* Grafana - [11.3.0](https://github.com/scylladb/scylla-monitoring/blob/4.8.3/versions.sh#L101)
* Prometheus - [2.54.1](https://github.com/scylladb/scylla-monitoring/blob/4.8.3/versions.sh#L99)
* Prometheus Operator - 0.78.2
* Bash tools image - UBI 9.5-1731517889
* Default dashboard version to ScyllaDB 6.2

**Which issue is resolved by this Pull Request:**
Resolves #2198

/cc

### CODE REVIEW

**Comment by zimnx** [comment]
Tested locally, dashboards in Grafana look ok.
/cc tnozicka 
/cc rzetelskik

File: assets/config/config.yaml [hunk 1]
```diff
@@ -8,9 +8,9 @@ operator:
   scyllaDBManagerVersion: "3.4.0@sha256:6424c36929b39ef4e679ee7d61578b1070e28c186b23b278147fde8e391d9db1"
   scyllaDBManagerAgentVersion: "3.4.0@sha256:441403aed8880cad1feef68aa7a8ee9ffd99a458dc1dcff3dc54ce5bf3cb07b7"
   bashToolsImage: "registry.access.redhat.com/ubi9/ubi:9.4-1214.1726694543@sha256:b00d5990a00937bd1ef7f44547af6c7fd36e3fd410e2c89b5d2dfc1aff69fe99"
```
[the remark this answers is not in the document]
Comment (zimnx) [reply assets/config/config.yaml]: good point, bumped to 9.5-1731517889

**Review by rzetelskik — APPROVED** [review]
lgtm
/assign tnozicka

**Pull request merged**